libnmstate/netapplier.py

```diff
@@ -360,9 +358,21 @@ def _sort_lag_slaves(desired_state, current_state):
 
 
 def _remove_absent_iface_entries(ifaces_desired_state):
```

Comment:
With the change, the method's name does not match it's purpose correctly anymore. Seems to me that it would be better to have two methods, one for absent interfaces and one for down virtual interfaces.

Reply:
I considered it an extension to the `absent`, because we handle both the same. When we will not handle them the same, we will not need to remove them as we do here.

Reply:
if you like to keep them in one method, then the method should be renamed to reflect that it also handles down interfaces. i.e. `_remove_down_virtual_and_absent_inface_entries()` which seems to be more ugly to me. But the current name is misleading now.

Reply:
Done